src/ui/src/js/controllers/command_index.js

```diff
@@ -29,12 +29,18 @@ export default function commandIndexController(
   detailSystems,
   commands) {
   $scope.setWindowTitle('commands');
-
+  $scope.filterHidden = false;
+  $scope.dtInstance={};
   $scope.dtOptions = DTOptionsBuilder.newOptions()
     .withOption('order', [4, 'asc'])
     .withOption('autoWidth', false)
     .withBootstrap();
 
```

Comment:
Extra whitespace

Reply:
I did not notice the overlapping because I did testing on one window size.